<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IconPanelClass</class>
 <widget class="QWidget" name="IconPanelClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_Button">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QRadioButton
{
	border-radius: 10px;
	font-family: 微软雅黑;
	font-size: 12px;
	color: rgba(0,0,0,1);
	padding-left:10px;
}

QRadioButton::checked
{
	font-family: 微软雅黑;
	font-size: 12px;
	color: #01EEC3;
}

QRadioButton::unchecked
{
	font-family: 微软雅黑;
	font-size: 12px;
	color: rgba(0,0,0,1);
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>10</number>
         </property>
         <property name="leftMargin">
          <number>16</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>16</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QRadioButton" name="radioButton1">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>160</width>
             <height>40</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>阿里巴巴图形字体</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton2">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>160</width>
             <height>40</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>国际知名图形字体</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton3">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>160</width>
             <height>40</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>天气图形字体</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButton4">
           <property name="minimumSize">
            <size>
             <width>160</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>160</width>
             <height>40</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>国际知名图形字体新</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <property name="styleSheet">
         <string notr="true">QWidget
{
background: transparent;
border:none;
}</string>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="PanelWidget" name="page1"/>
        <widget class="PanelWidget" name="page2"/>
        <widget class="PanelWidget" name="page3"/>
        <widget class="PanelWidget" name="page4"/>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PanelWidget</class>
   <extends>QWidget</extends>
   <header location="global">panelwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="IconPanel.qrc"/>
 </resources>
 <connections/>
</ui>
